src/test/java/uk/gov/pay/webhooks/queue/EventQueueIT.java

```diff
@@ -35,8 +34,8 @@ public void setUp() {
     }
 
     @Test
-    public void shouldReceiveMessageFromTheQueue() throws QueueException {
-        client.sendMessage(SqsTestDocker.getQueueUrl("event-queue"), "");
+    void shouldReceiveMessageFromTheQueue() throws QueueException {
+        client.sendMessage(SqsTestDocker.getQueueUrl("event-queue"), "{ messageBody: \"example message\" }");
```

Comment:
It's not required really, the test was failing because the message was 'empty' (presume something has changed to check for this in SQS client). 

This test is just checking that the app consumes from a queue, not how it deals with a message.